src/foam/nanos/menu/DAOMenu.js

```diff
@@ -16,6 +16,18 @@ foam.CLASS({
       class: 'String',
       name: 'daoKey'
     },
+    {
```

Comment:
We should probably add `searchMode` as an option here as well. That could be set through a journal file, it shouldn't require a custom controller model. Other properties on DAO Controller like `editEnabled` and `createEnabled` would be good too.

Reply:
Agreed.

Reply:
Resolved by 101ffa5.